build/yaml/cleanupResources/cleanupResources.yml

```diff
@@ -27,6 +27,7 @@ variables:
   InternalBotResourceGroupName: $[coalesce(variables['DEPLOYRESOURCEGROUP'], 'BFFN')]
   InternalSharedResourceGroupName: $[coalesce(variables['SHAREDRESOURCEGROUP'], 'BFFN-Shared')]
   InternalVirtualNetworkName: "bffnvirtualnetwork$($env:RESOURCESUFFIX)"
+  InternalStorageName: "bffnstorageaccount$($env:RESOURCESUFFIX)"
```

Comment:
I will rename this to InternalStorageAccountName to be more specific. If we decided to go this way, i change it from my side